string message
int32 a
int32 b
int16 v
int16 w
int32 send_CAN_counter
int32 get_CAN_counter
int32 set_speed_rpm
int32 set_torque_Nm
float64[] motor_rpm
float64[] motor_Nm
float64[] motor_break_code
float64[] motor_tempoC
float64[] motor_odom_er






